frontends/election-manager/src/app_root.tsx

```diff
@@ -218,6 +213,10 @@ export function AppRoot({
     [logger, currentUserRole, clearCastVoteRecordFilesMutation, store]
   );
 
+  if (!currentElection.isSuccess) {
+    return null;
+  }
```

Comment:
This actually fixed one of those "frontend restarted but backend didn't" bugs I referenced earlier. But unfortunately didn't fix them all

Reply:
Hah, I had left a comment in response to your comment about that suggesting exactly this fix. Would it be worth talking through the remaining issues?

Reply:
The main issue that remains without the logout on mount is the one that I mention in https://github.com/votingworks/vxsuite/pull/2960#discussion_r1092694177

Took some time to investigate further and one of the deeper problems is that, while election(s) are persisted in the admin service, the current election ID is still persisted in frontend local storage. This explains a lot of the funky behavior that I've seen when switching between incognito windows and election definitions

Would be open to more fully sharing my findings and discussing paths forward tomorrow!

Reply:
Ah interesting! Yes, let's discuss tomorrow. Though also if you want to move forward it doesn't necessarily need to block, since it's only a concern for dev

Reply:
I thought a bit more about this and decided to change course. All of the issues that remain have to do with multiple isolated browsers backed by the same backend service - a situation that only really comes up during Cypress tests. Rather than modifying source code to support that case, I decided to just update Cypress tests to explicitly log out before starting

With that, we now do get the handy benefit of not having to re-auth when the frontend restarts during development!

https://github.com/votingworks/vxsuite/pull/2960/commits/1dd74c8b0fbc628cb6b4b8d56528c230e9108668

Reply:
Awesome!